svc-bip-api/src/main/java/gov/va/vro/BipApplication.java

```diff
@@ -0,0 +1,24 @@
+package gov.va.vro;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.context.annotation.ComponentScan;
+
+@Slf4j
+@SpringBootConfiguration
+@ConfigurationPropertiesScan(basePackages = {"gov.va.vro.config"})
+@EnableAutoConfiguration
+@ComponentScan(
+    basePackages = {"gov.va.vro", "gov.va.starter.boot"},
+    excludeFilters = {
+
+    })
```

Comment:
Why not use `@SpringBootApplication` instead?

Reply:
Because I just copied/pasted from `:app` with the intention to change things as little as possible.  Are there good reasons to use`@SpringBootApplication` instead?

Reply:
Judge for yourself:
- https://docs.spring.io/spring-boot/docs/2.0.x/reference/html/using-boot-using-springbootapplication-annotation.html
- https://learnjava.co.in/the-springbootapplication-annotation-explained/
- https://www.java67.com/2018/05/difference-between-springbootapplication-vs-EnableAutoConfiguration-annotations-Spring-Boot.html